code-of-conduct/guide-code-of-conduct-incident-record.md

```diff
@@ -0,0 +1,98 @@
+  - `guide-code-of-conduct-incident-record.md`
+  - `guide-code-of-conduct-remediation-record`
+
+You might also consider using the `../our-team/template-our-team.md` template to let your community members know who they can contact to report a Code of Conduct violation. This document is not only necessary for Code of Conduct violations. It is a core document that helps you clearly communicate who belongs to your open source project or organization.
+
+
+## Why do I need a Code of Conduct incident record?
+
+See the `guide-code-of-conduct.md` for reasons to incorporate a Code of Conduct and response plan.
+
+The Code of Conduct incident record is part of the documentation that needs to be kept when investigating and resolving Code of Conduct incidents.
+It is important to file this documentation to enable the community moderators to identify and prevent potential repeated patterns of abuse in the community.
+
+
+## Content of the Code of Conduct incident record template
+
+The following sections provide guidance about how to fill out each section of the Code of Conduct incident record template.
+
+:information_source: The Code of Conduct incident record template includes boilerplate text that you can customize or adapt, use as-is, or completely replace with your own text.
+
+
+### About the "Introduction" section
+The *Introduction* section includes a description of the goals the community moderator should keep in mind while taking an incident report from an individual.
+The community moderator should refer to this section regularly to ensure that the incident reporter feels safe and supported throughout the process.
+
+
+### About the "Incident number" section
+
+Follow your organization's incident number protocol in assigning an incident number.
+
+The incident number can include the date an investigation was opened.
+For example, yyyy-001.
+
+
+### About the "Name of community moderator who took the report" section
+
+The community moderator who took the report lists their name here.
+
+
+### About the "Reporter's contact information" section
+
+This section is optional.
+If the reporter is comfortable giving their name and contact information, note that information here.
+
+
+### About the "Permission from incident reporter to proceed?" section
+
+Ask the incident reporter for permission to proceeed.
+Indicate their response in thise section.
+
+If an incident reporter does not give permission to proceed with an investigation, they will be given the option to hold the report “in escrow.”
+Escrowed reports will not be acted upon until there is a second report of the same incident or a similar incident involving the same individual.
+The goal of an escrow report is to retain a record of incidents in case there is a pattern of misbehavior by the same individual.
+
+If the reporter wishes to keep the report in escrow, the incident record should still be filled out and filed in the appropriate archives for future tracking.

```

Comment:
```suggestion
If the reporter wants to keep the report in escrow, the incident record should still be filled out and filed in the appropriate archives for future tracking.

```
suggestion: change wishes to wants. Subjective - I will raise as a style guide decision.

Reply:
Adopted.